connectors/spark/src/test/java/io/unitycatalog/spark/ManagedTableReadWriteTest.java

```diff
@@ -29,46 +29,52 @@
  */
 public abstract class ManagedTableReadWriteTest extends BaseTableReadWriteTest {
   private static final String DELTA_TABLE = "test_delta";
+  protected static final String TBLPROPERTIES_CLAUSE =
+      String.format(
+          "TBLPROPERTIES ('%s'='%s')",
+          UCTableProperties.CATALOG_MANAGED_KEY, UCTableProperties.CATALOG_MANAGED_VALUE);
 
   @Test
   public void testCreateManagedTableErrors() {
     session = createSparkSessionWithCatalogs(CATALOG_NAME);
     String fullTableName = CATALOG_NAME + "." + SCHEMA_NAME + "." + DELTA_TABLE;
-    assertThatThrownBy(() -> sql("CREATE TABLE %s(name STRING) USING parquet", fullTableName))
-        .hasMessageContaining("not support non-Delta managed table");
-    assertThatThrownBy(
-            () ->
-                sql(
-                    "CREATE TABLE %s(name STRING) USING delta TBLPROPERTIES ('%s' = 'disabled')",
-                    fullTableName, UCTableProperties.CATALOG_MANAGED_KEY))
-        .hasMessageContaining(
-            String.format("Should not specify property %s", UCTableProperties.CATALOG_MANAGED_KEY));
     assertThatThrownBy(
             () ->
                 sql(
-                    "CREATE TABLE %s(name STRING) USING delta TBLPROPERTIES ('%s' = 'disabled')",
-                    fullTableName, UCTableProperties.CATALOG_MANAGED_KEY_NEW))
-        .hasMessageContaining(
-            String.format(
-                "Should not specify property %s", UCTableProperties.CATALOG_MANAGED_KEY_NEW));
-    assertThatThrownBy(
-            () ->
-                sql(
-                    "CREATE TABLE %s(name STRING) USING delta TBLPROPERTIES ('%s' = 'some_id')",
-                    fullTableName, UCTableProperties.UC_TABLE_ID_KEY))
-        .hasMessageContaining(UCTableProperties.UC_TABLE_ID_KEY);
-    assertThatThrownBy(
-            () ->
-                sql(
-                    "CREATE TABLE %s(name STRING) USING delta TBLPROPERTIES ('%s' = 'some_id')",
-                    fullTableName, UCTableProperties.UC_TABLE_ID_KEY_OLD))
-        .hasMessageContaining(UCTableProperties.UC_TABLE_ID_KEY_OLD);
+                    "CREATE TABLE %s(name STRING) USING parquet %s",
+                    fullTableName, TBLPROPERTIES_CLAUSE))
+        .hasMessageContaining("not support non-Delta managed table");
+    for (String featureProperty :
+        List.of(UCTableProperties.CATALOG_MANAGED_KEY, UCTableProperties.CATALOG_MANAGED_KEY_NEW)) {
+      assertThatThrownBy(
+              () ->
+                  sql(
+                      "CREATE TABLE %s(name STRING) USING delta TBLPROPERTIES ('%s' = 'disabled')",
```

Comment:
can be a follow up PR in the interest of 0.3.1, but it seems create table SQL command is showing up in a lot of places and it might be a good idea to consolidate all of managed tables ones since they have this common theme of setting a property. might be easier in the future to fix stuff when we wont want the table property to be set.